src/plugins/data_view_field_editor/public/components/preview/field_list/field_list.tsx

```diff
@@ -217,7 +217,7 @@ export const PreviewFieldList: React.FC<Props> = ({ height, clearSearch, searchV
       ) : (
         <VirtualList
           className="eui-scrollBar"
-          style={{ overflowX: 'hidden' }}
+          css={{ overflowX: 'hidden' }}
```

Comment:
To update: `VirtualList`'s real name is `FixedSizeList`. This component accepts a `className` prop but the `overflowX` won't have the same precedence as before (check [source code](https://github.com/bvaughn/react-window/blob/master/src/createListComponent.js#L373)). 

Revert this line.